test/integration/alertsMenuView_integrationTests.js

```diff
@@ -13,11 +13,49 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-
 /*global sinon, todo, chai, describe, it, calledOnce*/
 //integration tests
-
 describe('alertsMenuView.js spec', function() {
+    var now_date = new Date();
+    var old_date = new Date();
+    old_date.setDate(old_date.getDate() - 2);
+
+    function formatDate(date) {
```

Comment:
Recommended to use moment.js for date manipulation.
http://momentjs.com/docs/#/manipulating/